docs/eigenda/integrations-guides/dispersal/dispersal-api/_category_.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I'm a bit confused about this category. If we are going to have this subcategory, then it seems like the following pages would be included:
- API error codes
- Serialization requirements
- Metering and rate limiting (arguable)

Reply:
Also, shouldn't this category be ordered first within the overall `dispersal` section?

Reply:
I'll nix it. The idea was to logically join these two documents in particular, since they cover the same general domain of how to interact with the API. The problem with actually joining the two documents is that the result would be very long. I don't feel strongly enough about this though so will put these two documents in the higher category.